web_pivot_computed_measure/tests/test_ui_pivot.py

```diff
@@ -10,18 +11,48 @@ class TestUIPivot(common.HttpCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+
+        # res_users_model_id = (
+        #     cls.env["ir.model"].search([("model", "=", "res.users")]).id
+        # )
+        # cls.env["ir.model.fields"].create(
+        #     {
+        #         "name": "x_user_year_born",
+        #         "model": "res.users",
+        #         "field_description": "Year Born",
+        #         "ttype": "integer",
+        #         "store": True,
+        #         "model_id": res_users_model_id,
+        #     }
+        # )
+        # cls.env["ir.model.fields"].create(
+        #     {
+        #         "name": "x_user_year_now",
+        #         "model": "res.users",
+        #         "field_description": "Year Now",
+        #         "ttype": "integer",
+        #         "store": True,
+        #         "model_id": res_users_model_id,
+        #     }
+        # )
+
```

Comment:
Just remove it?
```suggestion
```

Reply:
@chienandalu 

I am leaving my comment, as the original code is using a model and we get asked about the fields for this model.

We must create fake fields as while migrating another module errors came from the tests here.

user_year_now, and user_year_born fields

Check this PR please: #3037

Reply:
yeah, kinda makes sense without having an in depth look